src/core/utils/helpers/url.ts

```diff
@@ -200,3 +200,13 @@ export const isUrlValid = (text: string) => {
 export const currentLocationWithParametersUrl = (
   params: Record<string, string>
 ) => appendQueryParametersToUrl(new URL(getCurrentLocation()), params);
+
+export const getUrlHashWithPrefix = (prefix: string): string => {
```

Comment:
This is used twice. Once in `shouldOpenManifestationDetails` where it's not needed and once for `shouldOpenEditionsDisclosure` which is just interested in whether there is one. Maybe it should be a `hasMaterialUrlHash` (in another file, of course)?